.prettierrc

```diff
@@ -3,6 +3,7 @@
   "singleQuote": true,
   "printWidth": 80,
   "trailingComma": "all",
+  "endOfLine": "auto",
```

Comment:
This should go to a standalone commit. A new PR would be even better, if you don't mind.

Reply:
Done in #5673